Site/SiteConcentrateFileFinder.php

```diff
@@ -96,47 +90,4 @@ protected function getSiteDataFiles()
 
         return $files;
     }
-
-    protected function getDevelopmentDataFiles($include_dir)
```

Comment:
This was all dead code that was previously used to support development packages for a PEAR-package environment.